packages/desktop/components/popups/AddProposalPopup.svelte

```diff
@@ -40,7 +40,7 @@
                 nodeInput?.validate(),
             ])
             await registerParticipationWrapper()
-            updateActiveAccountMetadata($selectedAccount.index, {
+            updateSelectedAccountPersistedData($selectedAccount.index, {
```

Comment:
this function should update the selectedAccount, therefor the index is not needed

Reply:
or we name it `updateAccountPersistedData` and update the logic in that function (see comment there)

Reply:
reverted this change 🙏🏼